### PR TITLE
add doc for v5.1: Deploy -> Install TiKV

### PR DESCRIPTION
**Pull request by marsishandsome**
### What is changed?

TiKV's official website is planned to be refactored as discussed in https://github.com/tikv/website/issues/227.

This PR covers `Deploy ` -> `Install TiKV`

### Any related PRs or issues?

https://github.com/tikv/website/issues/227.

### Which version does your change affect?
v5.1

### CODE REVIEW

**Comment by marsishandsome** [comment]
@andylokandy @zz-jason  PTAL

File: content/docs/5.1/deploy/install/prerequisites.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        parent: Install TiKV
+        weight: 1
+---
+
+Typical deployments of TiKV include a number of components:
+
+* 3+ TiKV nodes
+* 3+ Placement Driver (PD) nodes
+* 1 Monitoring node
+* 1 or more client application
```
[the remark this answers is not in the document]
Comment (marsishandsome) [reply content/docs/5.1/deploy/install/prerequisites.md:16]: sure

**Comment by marsishandsome** [comment]
@zz-jason PTAL

File: content/docs/5.1/deploy/install/prerequisites.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
+---
+title: Prerequisites
+description: Learn the software and hardware recommendations for deploying and running TiKV
+menu:
+    "5.1":
+        parent: Install TiKV
+        weight: 1
+---
+As an open source distributed key-value database, TiKV can be deployed in the Intel architecture server, ARM architecture server, and major virtualization environments and runs well. TiKV supports most of the major hardware networks and Linux operating systems.
```
Comment (andylokandy) [inline content/docs/5.1/deploy/install/prerequisites.md:9]: I can't see the connection between `open source distributed key-value database` and cross-platform. May be better to split this sentence into two statements.

File: content/docs/5.1/deploy/install/prerequisites.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+| Software | Version         |
+|:-------- |:--------------- |
+| sshpass  | 1.06 or later   |
+| numa     | 2.0.12 or later |
+| tar      | any             |
+
+## Server recommendations
+
+You can deploy and run TiKV on the 64-bit generic hardware server platform in the Intel x86-64 architecture or on the hardware server platform in the ARM architecture. The requirements and recommendations about server hardware configuration (ignoring the resources occupied by the operating system itself) for development, test, and production environments are as follows:
```
Comment (andylokandy) [inline content/docs/5.1/deploy/install/prerequisites.md:46]:
```suggestion
You can deploy and run TiKV on the 64-bit generic hardware server platform in the Intel x86-64 architecture or on the hardware server platform in the ARM architecture. The recommendations about server hardware configuration (ignoring the resources occupied by the operating system itself) for development, test, and production environments are as follows:
```

File: content/docs/5.1/deploy/install/prerequisites.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+### Development and test environments
+
+| Component |   CPU   | Memory | Local Storage |       Network        | Instance Number (Minimum Requirement) |
+|:---------:|:-------:|:------:|:-------------:|:--------------------:|:-------------------------------------:|
+|    PD     | 4 core+ | 8 GB+  | SAS, 200 GB+  | Gigabit network card |                   1                   |
+|   TiKV    | 8 core+ | 32 GB+ | SAS, 200 GB+  | Gigabit network card |                   3                   |
```
Comment (andylokandy) [inline content/docs/5.1/deploy/install/prerequisites.md:53]: Why 3 TiKV is needed?

File: content/docs/5.1/deploy/install/prerequisites.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+{{< info >}}
+- It is strongly recommended to use higher configuration in the production environment.
+- It is recommended to keep the size of TiKV hard disk within 2 TB if you are using PCIe SSDs or within 1.5 TB if you are using regular SSDs.
+{{< /info >}}
+
+
+## Network requirements
+
+TiKV requires the following network port configuration to run. Based on the TiKV deployment in actual environments, the administrator can open relevant ports in the network side and host side.
```
Comment (andylokandy) [inline content/docs/5.1/deploy/install/prerequisites.md:76]:
```suggestion
TiKV uses the following network ports, and their default port numbers are listed below. Based on the TiKV deployment in actual environments, you can change the port number in the configuration.
```

File: content/docs/5.1/deploy/install/prerequisites.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+|     Component     | Default Port | Description                                                                             |
+|:-----------------:|:------------:|:--------------------------------------------------------------------------------------- |
+|       TiKV        |    20160     | the TiKV communication port                                                             |
+|       TiKV        |    20180     | the communication port to report TiKV status                                            |
```
Comment (andylokandy) [inline content/docs/5.1/deploy/install/prerequisites.md:81]:
```suggestion
|       TiKV        |    20180     | the port for fetching statistics, used by Prometheus                     |
```

File: content/docs/5.1/deploy/install/prerequisites.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+|        PD         |     2379     | the client port, for communication with clients                                         |
```
Comment (andylokandy) [inline content/docs/5.1/deploy/install/prerequisites.md:82]:
```suggestion
|        PD         |     2379     | the client port, entrance for the clients to connect TiKV cluster                                         |
```

File: content/docs/5.1/deploy/install/production.md [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+description: Deploy a TiKV Cluster for Production Using TiUP
+menu:
+    "5.1":
+        parent: Install TiKV
+        weight: 2
+---
+
+This guide describes how to install and deploy TiKV for production environment.
+
+[TiUP](https://github.com/pingcap/tiup) is a cluster operation and maintenance tool. TiUP provides [TiUP cluster](https://github.com/pingcap/tiup/tree/master/components/cluster), a cluster management component written in Golang. By using TiUP cluster, you can easily perform daily operations, including deploying, starting, stopping, destroying, scaling, and upgrading a TiKV cluster, and manage cluster parameters.
```
Comment (andylokandy) [inline content/docs/5.1/deploy/install/production.md:12]:
```suggestion
[TiUP](https://github.com/pingcap/tiup) is a cluster operation and maintenance tool. TiUP provides [TiUP cluster](https://github.com/pingcap/tiup/tree/master/components/cluster), a cluster management component written in Golang. By using TiUP cluster, you can easily do daily operations on a TiKV cluster, including deploying, starting, stopping, destroying, scaling, upgrading, and manage cluster parameters.
```

File: content/docs/5.1/deploy/install/verify.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
+---
+title: Verify Cluster Status
+description: Check the cluster status and connect to the cluster
+menu:
+    "5.1":
+        parent: Install TiKV
+        weight: 3
+---
+
+After a TiKV cluster is deployed, you need to check whether the cluster runs normally. This document introduces how to check the cluster status using TiUP commands and Grafana, and how to connect to the TiKV cluster to perform simple `put` and `get` operations.
```
Comment (andylokandy) [inline content/docs/5.1/deploy/install/verify.md:10]:
```suggestion
After a TiKV cluster is deployed, you need to check whether the cluster is up and running. This document will introduce the methods to check the cluster status using TiUP and Grafana, and will also describe how to connect to the TiKV cluster using a TiKV client to do simple `put` and `get` operations.
```

File: content/docs/5.1/deploy/install/verify.md [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+
+### Use Grafana
+
+1. Log in to the Grafana monitoring at `${Grafana-ip}:3000`. The default username and password are both `admin`.
+
+2. To check the TiKV port status and load monitoring information, click **Overview**.
+
+## Connect to TiKV and perform simple operations
+
+This section describes how to connect to the TiKV cluster to perform simple `put` and `get` operations.
```
Comment (andylokandy) [inline content/docs/5.1/deploy/install/verify.md:34]:
```suggestion
This section describes how to connect to the TiKV cluster to do simple `put` and `get` operations.
```

File: content/docs/5.1/deploy/install/verify.md [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+
+Expected output: If the `Status` information of each node is `Up`, the cluster runs normally.
+
+### Use Grafana
+
+1. Log in to the Grafana monitoring at `${Grafana-ip}:3000`. The default username and password are both `admin`.
+
+2. To check the TiKV port status and load monitoring information, click **Overview**.
+
+## Connect to TiKV and perform simple operations
```
Comment (andylokandy) [inline content/docs/5.1/deploy/install/verify.md:32]:
```suggestion
## Connect to TiKV and do simple operations
```

File: content/docs/5.1/deploy/install/prerequisites.md (hunk 2 above)
Comment (marsishandsome) [reply content/docs/5.1/deploy/install/prerequisites.md:9]: sure

File: content/docs/5.1/deploy/install/prerequisites.md (hunk 4 above)
Comment (marsishandsome) [reply content/docs/5.1/deploy/install/prerequisites.md:53]: At least 3 TiKV is needed for production environments.

Comment (marsishandsome) [reply content/docs/5.1/deploy/install/prerequisites.md:54]: For performance-related test, the benchmark will be inaccurate if using only one TiKV.

**Review by andylokandy — APPROVED** [review]

**Review by zz-jason — APPROVED** [review]
LGTM

**Pull request merged**